Source/WebCore/page/LocalFrameView.h

```diff
@@ -167,6 +167,7 @@ class LocalFrameView final : public FrameView {
     void styleAndRenderTreeDidChange() override;
     bool updateCompositingLayersAfterStyleChange();
     void updateCompositingLayersAfterLayout();
+    bool updateCompositingLayersAfterLayoutIfNeeded();
```

Comment:
Without a comment, I am not sure it’s obvious from the name that the `bool` return value is a “did work” indicator.